PWGGA/GammaConvBase/AliConvEventCuts.h

```diff
@@ -671,6 +669,7 @@ class AliConvEventCuts : public AliAnalysisCuts {
       TString*                    fGeneratorNames;                        //[fnHeaders]
       PeriodVar                   fPeriodEnum;                            ///< period selector
       EnergyVar                   fEnergyEnum;                            ///< energy selector
+      AliTimeRangeCut             fTimeRangeCut; 
```

Comment:
please use the streamer //! or update the classdef

Reply:
okay, forgot the clasdef again. What is the streamer //! ? How would that work?